src/components/transactions/AssetInput.tsx

```diff
@@ -121,35 +124,41 @@ export const AssetInput = <T extends Asset = Asset>({
         })}
       >
         <Box sx={{ display: 'flex', alignItems: 'center', mb: 0.5 }}>
-          <InputBase
-            sx={{ flex: 1 }}
-            placeholder="0.00"
-            disabled={disabled || disableInput}
-            value={value}
-            autoFocus
-            onChange={(e) => {
-              if (!onChange) return;
-              if (Number(e.target.value) > Number(maxValue)) {
-                onChange('-1');
-              } else {
-                onChange(e.target.value);
-              }
-            }}
-            inputProps={{
-              'aria-label': 'amount input',
-              style: {
-                fontSize: '21px',
-                lineHeight: '28,01px',
-                padding: 0,
-                height: '28px',
-                textOverflow: 'ellipsis',
-                whiteSpace: 'nowrap',
-                overflow: 'hidden',
-              },
-            }}
-            // eslint-disable-next-line
-            inputComponent={NumberFormatCustom as any}
-          />
+          {loading ? (
+            <Box sx={{ flex: 1 }}>
+              <CircularProgress color="inherit" size="16px" />
+            </Box>
```

Comment:
Should we consider using a skeleton input here?

Reply:
I think loading spinner was part of the original design. I guess I'd lean towards just keeping it as is for now.